docs/conf/eu/2016/news/welcome.md

```diff
@@ -0,0 +1,37 @@
+:template: 2016/eu.html
+
+# Welcome to Write the Docs
+
+Hello Documentarians,
+
+The conference is just a few days away.
+We have some reminders, late-breaking information, and a few thanks are in order.
+
+We hope you're getting excited!
+
+## Info
+
+* If you have signed up for the **boat trip**, join us on Saturday the 17th at Prague Boats,
+  [pier no. 5 of Dvořákovo nábřeží](https://goo.gl/maps/bqLP3VaytVo). Boarding is
+  at 13:45, don't be late! The trip lasts for two hours, and will feature an  
+  English-speaking guide.
+* **Join us on Sunday** for the [Writing Day](http://www.writethedocs.org/conf/eu/2016/writingday/) from *09:00 to 17:00*, immediately followed by the reception from *17:00-19:00*. You can pick up your badge and enjoy a couple of drinks on us. This is a great chance to meet your fellow documentarians before the conference starts.
```

Comment:
please add that the writing day includes breakfast and lunch, and point them to the schedule for details :)